settings.gradle

```diff
@@ -0,0 +1 @@
+rootProject.name = 'beats'
```

Comment:
I dont have a strong opinion on this file, its was created by intellij when scaffolding a new gradle project. So I can remove it